Python/errors.c

```diff
@@ -52,6 +52,14 @@ PyErr_Restore(PyObject *type, PyObject *value, PyObject *traceback)
     Py_XDECREF(oldtraceback);
 }
 
+PyExcState *_PyErr_GetExcInfo(PyThreadState *tstate) {
+    PyExcState *exc_info = tstate->exc_info;
+    while ((exc_info->exc_type == NULL || exc_info->exc_type == Py_None) &&
+           exc_info->exc_previous != NULL)
```

Comment:
Another nit, per PEP 7 we now want to use `{` even for blocks with one statement.